invenio_deposit/ext.py

```diff
@@ -58,6 +61,11 @@ def init_config(self, app):
             if k.startswith('DEPOSIT_'):
                 app.config.setdefault(k, getattr(config, k))
 
+    @staticmethod
+    def register_signals():
+        """Register signals."""
+        post_action.connect(index_record_after_publish)
```

Comment:
I think the idea is to re-index the deposit with its published record like: https://github.com/inveniosoftware/invenio-deposit/blob/master/invenio_deposit/views/rest.py#L199